README.md

```diff
@@ -522,10 +522,36 @@ ansible-playbook install-golem.yml                                 \
 `golem_version` parameter determines which branch/tag/commit from the `golem` repository will be deployed in the machine.
 Version listed in `containers/versions.yml` in `concent-deployment` repository is used by default.
 
+##### Installing Golem GUI
+After installing Golem you may opt to install its `golem-electron` GUI too.
+The playbook also installs a lightweight XFCE desktop environment in the machine that lets you actually run it inside without connecting to the X server running on the host.
+
+To install it, run the following playbook:
+``` bash
+ansible-playbook install-golem-gui.yml                             \
+    --extra-vars  golem_electron_version=dev                       \
+    --private-key .vagrant/machines/default/virtualbox/private_key \
+    --user        vagrant                                          \
+    --inventory   inventory
+```
+
+`golem_electron_version` parameter determines which branch/tag/commit from the `golem-electron` repository will be deployed in the machine.
+The default is the `dev` branch.
+
+To be able to run graphical programs inside the machine, you also have to perform some manual steps:
+- Make sure the machine is not running.
+    If it is, stop it with `vagrant halt`.
+- Start VirtualBox GUI.
+- Select the `concent-vm` image.
+- Go to Settings > Display.
+- Set video memory to 128 MB.
```

Comment:
Please be more specific here. You should add information that after this step the virtual machine should be running by `vagrant up`

Reply:
I have added a sentence saying that you can go to VirtualBox GUI and show the screen. It's then removed in `chore-automate-and-simplify-concent-vm-setup` branch because that one already says that the GUI starts automatically.